tests/integration/cmd_devfile_push_test.go

```diff
@@ -0,0 +1,156 @@
+package integration
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/openshift/odo/tests/helper"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("odo devfile push command tests", func() {
+	var project string
+	var context string
+	var currentWorkingDirectory string
+
+	var sourcePath = "/projects"
+
+	oc = helper.NewOcRunner("oc")
+
+	// This is run after every Spec (It)
+	var _ = BeforeEach(func() {
+		SetDefaultEventuallyTimeout(10 * time.Minute)
+		project = helper.CreateRandProject()
+		context = helper.CreateNewContext()
+		currentWorkingDirectory = helper.Getwd()
+		helper.Chdir(context)
+		os.Setenv("ODO_EXPERIMENTAL", "true")
+	})
+
+	// Clean up after the test
+	// This is run after every Spec (It)
+	var _ = AfterEach(func() {
+		helper.DeleteProject(project)
+		helper.DeleteDir(context)
```

Comment:
It will throw error on windows platform. Rearranging the line with the below one will fix the problem

```
helper.Chdir(currentWorkingDirectory)
helper.DeleteDir(context)
```

Reply:
Ahh, that's a good point about Windows. Updated.